ts/DataGrid/Accessibility/Accessibility.ts

```diff
@@ -85,6 +86,8 @@ class Accessibility {
         this.announcerElement = document.createElement('p');
         this.announcerElement.setAttribute('aria-atomic', 'true');
         this.announcerElement.setAttribute('aria-hidden', 'false');
+
+        this.addHighContrast();
```

Comment:
This method should not be called in the a11y constructor, but during DataGrid rendering. Currently, for example, the high contrast theme is removed after update.

This logic can be added to `initTableA11yAttrs`, I would also change the name of this method to something more general.